test_runner/src/main/kotlin/ftl/ios/xctest/common/InstallParseBinaries.kt

```diff
@@ -40,6 +40,7 @@ private fun neededFilesListByOs(): List<String> = if (isWindows) {
 private val flankBinariesDirectory = Paths.get(appDataDirectory, ".flank").toAbsolutePath()
 
 private fun downloadAndUnzip(osname: String) {
+    val dummyVar = 1
```

Comment:
```suggestion
    val dummyVar = ""
    print(dummyVar)
```

This should prevent build fails

Reply:
Thanks hahaha